index.html

```diff
@@ -79,12 +79,23 @@ <h2>CapturedMouseEvent interface</h2>
       <pre class="idl">
         [Exposed=Window]
         interface CapturedMouseEvent : Event {
-          constructor(optional CapturedMouseEventInit eventInitDict = {});
+          constructor(DOMString type, optional CapturedMouseEventInit eventInitDict = {});
           readonly attribute long surfaceX;
           readonly attribute long surfaceY;
         };
       </pre>
       <dl data-link-for="CapturedMouseEvent" data-dfn-for="CapturedMouseEvent">
+        <dt><dfn constructor>constructor()</dfn></dt>
+        <dd>
+          <p>Constructs a new {{CapturedMouseEvent}}.</p>
+          <p>The arguments are passed as-it to {{Event}}'s constructor.</p>
```

Comment:
Typo with "as-it", probably meant "as-is". But possibly this should be plural given that multiple arguments are involved?

Reply:
Yes should be "as is" and I believe it's invariable.